src/main/java/org/opensearch/ad/model/DetectorValidationIssueType.java

```diff
@@ -24,7 +24,10 @@
     FILTER_QUERY(AnomalyDetector.FILTER_QUERY_FIELD),
     WINDOW_DELAY(AnomalyDetector.WINDOW_DELAY_FIELD),
     GENERAL_SETTINGS(AnomalyDetector.GENERAL_SETTINGS),
-    RESULT_INDEX(AnomalyDetector.RESULT_INDEX_FIELD);
+    RESULT_INDEX(AnomalyDetector.RESULT_INDEX_FIELD),
+    GENERAL_DATA(AnomalyDetector.GENERAL_DATA),
```

Comment:
Can you explain what this issue type means?

Reply:
I'll actually change this since it occurs when `lastTimestamp` produces a negative value which means there isn't enough historical data with the timestamp used to get the time range. But this is just the effect and not the cause which is a timefield being chosen with no latest document found for it. Will change message to convey that and remove general_data issue type for now.

Reply:
Removed this field and changed the use of it to be referring to the timefield since the issue leads to no historical data but it cause no last timestamp can be found and this is after the check that there is at least 1 document in index